server/api/v1/event/[eventSlug]/subscription/mine.get.ts

```
@@ -13,5 +22,28 @@ export default defineEventHandler(async (event) => {
+    const parsed = querySchema.safeParse(query);
+
+    if (!parsed.success) {
+      throw createError({
+        statusCode: 422,
+        statusMessage: "Invalid request data",
+        data: parsed.error.errors,
+      });
+    }  
+   
+    subscription.lastQuestionnairePreScreening = {
+      formResponseId: new Types.ObjectId(parsed.data.formResponseId) as unknown as {
```

ESLint

[error] Replace `parsed.data.formResponseId` with `⏎········parsed.data.formResponseId,⏎······`

(prettier/prettier)